biorxiv/README.md

```diff
@@ -0,0 +1,13 @@
+# Directory Structure
+
+| Folder/file | Description |
```

Comment:
Based on this description it doesn't seem like there is an order to running the notebooks in these different folders. Is that correct. If not, you might want to specify this somewhere

Reply:
>Based on this description it doesn't seem like there is an order to running the notebooks in these different folders. Is that correct. If not, you might want to specify this somewhere 

That is correct. There isn't a specified order to run notebooks.